<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SPL1281" ignored="0" name="Default test" tests="4" failures="0" timestamp="24 Oct 2021 06:39:08 GMT" time="0.033" errors="0">
  <testcase name="@BeforeMethod setUp" time="0.004" classname="com.crm.qa.testcases.ContactsPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.crm.qa.base.TestBase.initialization(TestBase.java:41)
	at com.crm.qa.testcases.ContactsPageTest.setUp(ContactsPageTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="verifyContactsPageLable" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- verifyContactsPageLable -->
  <testcase name="selectSingleContactTest" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- selectSingleContactTest -->
  <testcase name="selectMultipleContactTest" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- selectMultipleContactTest -->
  <testcase name="validateCreateNewContact" time="0.0" classname="com.crm.qa.testcases.ContactsPageTest">
    <skipped/>
  </testcase> <!-- validateCreateNewContact -->
</testsuite> <!-- Default test -->
